static/js/lib/api.js

```diff
@@ -92,7 +92,7 @@ import { fetchWithCSRF } from './api';
  *  - non 2xx status codes will reject the promise returned
  *  - response JSON is returned in place of response
  */
-const _fetchJSONWithCSRF = (input: string, init: Object = {}, loginOnError: boolean = false): Promise<*> => {
+const _fetchJSONWithCSRF = (input: string, init: Object = {}, loginOnError: boolean = true): Promise<*> => {
```

Comment:
I think we should only default to true for GET requests. 400 is returned whenever there is a validation error for POST requests. Can we leave it as `false` then deal with it in the next PR?